src/UiPath.CoreIpc.Tests/Implementation/SystemService.cs

```diff
@@ -175,5 +180,27 @@ public async Task<Stream> Echo(Stream input, CancellationToken cancellationToken
             result.Position = 0;
             return result;
         }
+
+        [DllImport("kernel32.dll", SetLastError = true)]
+        public static extern bool CancelIoEx(IntPtr handle, IntPtr lpOverlapped);
+        public async Task<bool> CancelIoPipe(Message message = null, CancellationToken cancellationToken = default)
+        {
+            await Task.Delay(10);
+#if WINDOWS
+            var networkPropertyInfo = message.Client.GetType().GetProperty("Network", BindingFlags.NonPublic | BindingFlags.Instance);
```

Comment:
we would not use reflection for this on develop

Reply:
Ok, for develop